cmd/thanos/store.go

```diff
@@ -359,6 +359,13 @@ func parseRelabelConfig(contentYaml []byte) ([]*relabel.Config, error) {
 	if err := yaml.Unmarshal(contentYaml, &relabelConfig); err != nil {
 		return nil, errors.Wrap(err, "parsing relabel configuration")
 	}
+	supportedActions := map[relabel.Action]struct{}{relabel.Keep: {}, relabel.Drop: {}, relabel.HashMod: {}}
+
+	for _, cfg := range relabelConfig {
+		if _, isPresent := supportedActions[cfg.Action]; !isPresent {
```

Comment:
nit: We usually use `ok` as pattern here. Let's be consistent, although `isPresent` is really nice as well.
```suggestion
		if _, ok := supportedActions[cfg.Action]; !ok {
```